packages/react-components/src/components/Picker/Picker.tsx

```diff
@@ -129,23 +201,28 @@ export const Picker: React.FC<IPickerProps> = ({
           isError={!!error}
           isOpen={isListOpen}
           isDisabled={disabled}
-          isItemSelected={!!selectedItem}
+          isItemSelected={!!selectedItems}
           isRequired={isRequired}
+          isMultiSelect={multiselect}
           size={size}
           onClick={handleOnTriggerClick}
           onClearClick={handleOnClearClick}
           onFilter={handleOnFilter}
         >
-          {selectedItem ? selectedItem.name : placeholder}
+          {selectedItems ? getSelectedItems(selectedItems) : placeholder}
         </Trigger>
         <PickerList
-          selectedItem={selectedItem}
+          selectedItemsKeys={
+            selectedItems ? getSelectedItemsKeys(selectedItems) : null
```

Comment:
And again, is operating on `null` necessary. `undefined` or empty array would probably be enough and would bring fewer issues in the code.

Reply:
As above